src/components/dialog/dialog.pw.tsx

```diff
@@ -357,6 +358,22 @@ test.describe("Testing Dialog component properties", () => {
       page.getByText("I should not be scrolled into view")
     ).not.toBeInViewport();
   });
+
+  test("should loop focus whenwhen a Select component is passed as children and the user presses shift + tab", async ({
```

Comment:
suggestion: double when

```suggestion
  test("should loop focus when a Select component is passed as children and the user presses shift + tab", async ({
```